help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -0,0 +1,192 @@
+
+The list of attribute mappings that can be set-up for the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts) is given below. The destination uses the [Salesforce Search Attribute-Set Definitions REST API](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/retrieveAttributeSetDefinitions.html) to retrieve attributes defined within Salesforce for your contacts and specific to your account. 
+
+> [!IMPORTANT]
+> 
+> Although your attribute names would be as per your Salesforce account, the mappings for `contactKey` and `personalEmail.address` are mandatory.
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select attributes]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/source-mapping.png)
+
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Target mapping](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/target-mapping.png)
+
+1. To map any custom attributes select target field window, select the target field and choose the **[!UICONTROL Select attributes]** > **Email Demographics** category, Next provide the desired target attribute name and add the mappings desired.
+![Target mapping](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/target-mapping-custom.png)
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Salesforce Marketing Cloud] instance:
+
+    ||XDM Profile Schema|[!DNL Salesforce Marketing Cloud] Instance| Mandatory|
+    |---|---|---|---|
+    |Attributes|<ul><li><code>person.name.firstName</code></li><li><code>personalEmail.address</code></li></ul>|<ul><li><code>Email Demographics.First Name</code></li><li><code>Email Addresses.Email Address</code></li></ul>|<ul><li>-</li><li><code>Yes</code></li></ul>|
+    |Identities|<ul><li><code>contactKey</code></li></ul>|<ul><li><code>salesforceContactKey</code></li></ul>|Yes|
+
+1. An example using these mappings is shown below:
+![Target mapping LastName](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/mappings.png)
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you will need to manually map Platform segments to the custom attribute in Salesforce.
```

Comment:
Same comment as for the other Salesforce destination that you are working on: https://github.com/AdobeDocs/experience-platform.en/pull/60/files#r921524882